pkg/apis/kops/componentconfig.go

```diff
@@ -420,6 +420,15 @@ type KubeAPIServerConfig struct {
 	// otherwise the host's root CA set will be used.
 	OIDCCAFile *string `json:"oidcCAFile,omitempty" flag:"oidc-ca-file"`
 
+	// If set, all requests coming to the apiserver will be logged to this file.
+	AuditLogPath *string `json:"auditLogPath,omitempty" flag:"audit-log-path"`
+	// The maximum number of days to retain old audit log files based on the timestamp encoded in their filename.
+	AuditLogMaxage *string `json:"auditLogMaxage,omitempty" flag:"audit-log-maxage"`
```

Comment:
So I think maxage, maxbackups and maxsize are all int (path is a string).  Could I ask you to change them to `int32`s?

```
staging/src/k8s.io/apiserver/pkg/server/options/audit.go:       fs.StringVar(&o.Path, "audit-log-path", o.Path,
staging/src/k8s.io/apiserver/pkg/server/options/audit.go:       fs.IntVar(&o.MaxAge, "audit-log-maxage", o.MaxBackups,
staging/src/k8s.io/apiserver/pkg/server/options/audit.go:       fs.IntVar(&o.MaxBackups, "audit-log-maxbackup", o.MaxBackups,
staging/src/k8s.io/apiserver/pkg/server/options/audit.go:       fs.IntVar(&o.MaxSize, "audit-log-maxsize", o.MaxSize,
```